integrations/auth_ldap_test.go

```diff
@@ -112,7 +112,7 @@ func addAuthSourceLDAP(t *testing.T, sshKeyAttribute string) {
 		"attribute_name":           "givenName",
 		"attribute_surname":        "sn",
 		"attribute_mail":           "mail",
-		"attribute_ssh_public_key": sshKeyAttribute,
+		"attribute_ssh_public_key": "sshPublicKey",
```

Comment:
why this change? This was intentionally to have option to test different scenarious

Reply:
Well, you're right. Even if it was unnecessary, it's out of scope for this PR.